test/chroner/administrative_test.exs

```diff
@@ -0,0 +1,141 @@
+defmodule Chroner.AdministrativeTest do
+  use ExUnit.Case, async: false
+  use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
+
+  import Chroner.Administrative
+
+  alias Chroner.Administrative.{Doctor, User}
+  alias OAuth2.Response
+
+  setup do
+    valid_client = Chroner.client(access_token: System.get_env("ACCESS_TOKEN"))
+    invalid_client = Chroner.client(access_token: "INVALID_TOKEN")
+    {:ok, %{valid_client: valid_client, invalid_client: invalid_client}}
+  end
+
+  # --------------------------------------------------------------------
+  # Doctors
+  # --------------------------------------------------------------------
+
+  describe "doctors_list/1" do
+    test "gets all doctors", %{valid_client: client} do
+      use_cassette "doctors_list_success" do
+        assert {:ok, [%Doctor{} | _]} = doctors_list(client)
```

Comment:
Could we get a more precise test, e.g. by verifying the expected ids of all doctors? The same remark holds for other tests.

Reply:
It's hard to do this, because the state on the test app changes all the time. I actually don't what to expect here besides list of all Doctors in the system.

Reply:
State of which app? Doesn't this depend only on the content of the recorded cassettes?